content/docs/userspace/http-api-guide.md

```diff
@@ -0,0 +1,649 @@
+| `threadName` | `string` | The name of the thread to run.                                                                                | `"graph-eval"`          |
+| `body`       | any JSON | The data to give to the thread as its argument.                                                               | `{foo: "bar", baz: 42}` |
+| `desk`       | `string` | The desk in which the thread resides. This may be ommitted if previously set for the whole `Urbit` object.    | `"landscape"`           |
+
+The `thread` function will produce a promise that, if successful, contains the
+JSON result of the thread.
+
+#### Example
+
+This example takes a hoon expression (such as `(add 1 1)`), evalutes it with the
+`graph-eval` thread in the `landscape` desk, and prints the result.
+
+```
+<html>
+  <head>
+    <script src="https://unpkg.com/@urbit/http-api"></script>
+    <script src="/session.js"></script>
+  </head>
+  <body>
+    <input id="hoon" type="text" placeholder="Hoon to evaluate" />
+    <button id="submit" type="button" onClick="runThread()" >Submit</button>
+    <pre id="expr"></pre>
+    <pre id="result"></pre>
+  </body>
+  <script>
+    document.getElementById("hoon")
+      .addEventListener("keyup", function(event) {
+        if (event.keyCode === 13) {
+          document.getElementById("submit").click();
+        }
+      });
+    const hoon = document.getElementById("hoon");
+    const expr = document.getElementById("expr");
+    const result = document.getElementById("result");
+    const desk = "landscape";
+    const api = new UrbitHttpApi.Urbit("", "", desk);
+    api.ship = window.ship;
+    async function runThread() {
+      const threadResult = await api.thread({
+        inputMark: "graph-view-action",
+        outputMark: "tang",
+        threadName: "graph-eval",
+        body: {eval: hoon.value}
+      });
+      result.innerHTML = threadResult[0].join("\n");
+      expr.innerHTML = hoon.value;
+      hoon.value = "";
+    };
+  </script>
+</html>
+```
+
+### Delete Channel
+
+Rather than just closing individual subscriptions, the entire channel can be
+closed with the `delete` function in the `Urbit` class of `http-api`. The
+function takes no arguments, and can be called like `api.cancel()` (replacing
+`api` with whatever you've named the `Urbit` object you previously instantiated).
+When a channel is closed, all subscriptions will be cancelled and all pending
+updates will be discarded.
+
+### Reset
+
+An existing instance of `http-api`'s `Urbit` class can be reset with its
+`reset` function. This function takes no arguments, and can be called like
+`api.reset()` (replacing `api` with whatever you've named the `Urbit` object you
+previously instantiated). When called, the channel will be closed and all
+subscriptions cancelled. Additionally, all outstanding outbound pokes will be
+discarded and a fresh channel ID will be generated.
+
+## Further reading
```

Comment:
Any particularly exemplary instances we can suggest the interested to investigate?  Ideally ones that are simpler than graph-store, so not Landscape or EScape.